actors/builtin/market/market_state.go

```diff
@@ -289,13 +318,23 @@ func (st *State) maybeLockBalance(rt Runtime, addr addr.Address, amount abi.Toke
 
 // TODO: all these balance table mutations need to happen at the top level and be batched (no flushing after each!)
 // https://github.com/filecoin-project/specs-actors/issues/464
-func (st *State) unlockBalance(lt *adt.BalanceTable, addr addr.Address, amount abi.TokenAmount) error {
+func (st *State) unlockBalance(lt *adt.BalanceTable, addr addr.Address, amount abi.TokenAmount, lockReason BalanceLockingReason) error {
 	Assert(amount.GreaterThanEqual(big.Zero()))
 
 	err := lt.MustSubtract(addr, amount)
 	if err != nil {
 		return xerrors.Errorf("subtracting from locked balance: %v", err)
 	}
+
+	switch lockReason {
+	case ClientCollateral:
+		st.TotalLockedClientCollateral = big.Sub(st.TotalLockedClientCollateral, amount)
+	case ClientStorageFee:
+		st.TotalLockedDealPayments = big.Sub(st.TotalLockedDealPayments, amount)
```

Comment:
Unify the terminology here between the reason and the state variable. I suggest `st.TotalClientStorageFee`.

And just for super consistency, how about `TotalClientLockedCollateral` and `TotalProviderLockedCollateral` too.

Reply:
This is done.